customization/dispvm-customization.md

```diff
@@ -57,6 +57,42 @@ It is possible to change the settings for each new Disposable VM (DispVM). This
 4.  Shutdown the qube (either by `poweroff` from qube's terminal, or `qvm-shutdown` from dom0 terminal).
 
 
+### Using static Disposable VMs for sys-* ###
+
+You can use a static DispVM for `sys-*` as long as it is stateless.
+For example, a `sys-net` using DHCP or `sys-usb` will work.
+`sys-firewall` will not work unless you have no custom rules set, because per VM rules are stored in a configuration file inside that AppVM.
```

Comment:
I'd phrase it the other way around. Right now it suggest stateless sys-firewall is rare case, which I think is not true - custom rules are rare case.